app/Http/Controllers/Admin/CustomerCrudController.php

```diff
@@ -54,19 +54,15 @@ protected function setupListOperation()
 
         $this->crud->modifyColumn('signature', [
             'type' => 'image',
-            'height' => '100px',
-            'width'  => '100px',
+            'height' => '150px',
+            'width'  => '150px',
         ]);
     }
 
+
     protected function setupShowOperation()
     {
         $this->setupListOperation();
-
-        $this->crud->modifyColumn('photo', [
-            'height' => '150px',
-            'width'  => '140px',
-        ]);
```

Comment:
@chrisjimghub para ni siya sa preview bai. kung imo gusto is same ra ang size sa datatable row photo og sa preview, then ok. kung gamyan gani ka. e modify lng sa showOperation.
ang showOperation method mao ng preview nga button